add-on/src/options/forms/gateways-form.js

```diff
@@ -31,6 +31,7 @@ export default function gatewaysForm ({
   const onDisabledOnChange = onOptionChange('disabledOn', hostTextToArray)
   const onEnabledOnChange = onOptionChange('enabledOn', hostTextToArray)
   const mixedContentWarning = !secureContextUrl.test(customGatewayUrl)
+  // TODO(whizzzkid): remove this once we have a better solution
   const supportRedirectToCustomGateway = ipfsNodeType !== 'embedded'
```

Comment:
this will need to be fixed before merging Mv3 right?

What are the existing node types? Only 'brave' and 'remote' ?